### ISSUE TITLE
Kubeadm tests have been failing for days. 

### ISSUE THREAD

**Issue by timothysc** [kubernetes/kubernetes#68475]
**Is this a BUG REPORT**:
https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-all#kubeadm-gce-1.12  & 
https://k8s-testgrid.appspot.com/sig-cluster-lifecycle-all#kubeadm-gce-master 

Have been failing since we PR'd job changes to test-infor

/kind bug

**What happened**:
kubernetes-anywhere deployment appears to be failing on CNI deployment of weave in the test. 

Here is what happens: 
init of master (success) 
join of nodes (success) 
deployment of weave (success) 
verification of nodes ready (fail) 
 - On inspection of the nodes I do not see CNI pods getting started. 

It's difficult to diagnose, b/c I do not see the api-server+controller-manager+scheduler logs to diagnose

**What you expected to happen**:
non-failure 

/cc @kubernetes/sig-cluster-lifecycle @kubernetes/test-infra-admins @kubernetes/release-admins  @neolit123 @fabriziopandini  

Right now I'm totally blocked on other work, so I need to hand-off some of the investigations

**Comment by mohammedzee1000** [comment]
@neolit123 thanks for update. Do you want to use this issue to track, or open new issue

**Comment by neolit123** [comment]
@mohammedzee1000 
@timothysc opened this one so we are going to use it instead of:
https://github.com/kubernetes/kubernetes/issues/66338

**Comment by errordeveloper** [comment]
cc @bboreham @brb @murali-reddy

**Comment by timothysc** [comment]
We're pretty sure it's due to the older version of weave being used in kubernetes anywhere. 

```
 @timothysc with the old weave my single master 1.12 cluster did ready-up, but the weave-net pod is a 
`CrashLoopBackOff`. i guess we should bump the weave-net version in k-a anyhow.
```

**Comment by bboreham** [comment]
On Slack I was pointed at a log file https://pastebin.com/6kzwsRmS, which has the same symptom as http://github.com/weaveworks/weave/issues/2939 

```
FATA: 2018/09/10 21:29:27.302673 ipset [destroy weave-local-pods] failed: ipset v6.29: Set cannot be destroyed: it is in use by a kernel component
```

we never got to the bottom of it.

Is the node a freshly-created VM?

**Comment by neolit123** [comment]
@bboreham 
> Is the node a freshly-created VM?

no, it's a VM that is clobbered for various kubeadm based control plane node setups.

/lifecycle active

**Comment by neolit123** [comment]
both tests are green now and we will watch them closely until release. :tada: 

@boreham from what i saw i couldn't figure out if a process has exclusive access. my immediate answer is "no" but i might have missed something.
one e2e and two different people testing 1.9.4 on latest kubernetes master were able to reproduce this on Ubuntu, so you might be able to debug yourself further?

using 2.4.0 looks like a definitive fix for us.
thanks!

/close
